apps/calc/src/App.tsx

```diff
@@ -1,12 +1,11 @@
-import React, { useEffect } from "react";
+import { useEffect } from "react";
 import { BrowserRouter, HashRouter } from "react-router-dom";
 import i18n from "./i18n/index";
-import { ThemeProvider } from "styled-components";
 import { useSelector } from "react-redux";
-import GlobalStyle from "@components/global-styles/global-styles";
 import AppRoutes from "@local-components/app-routes/app-routes";
-import AppHeaderNavBar from "@local-components/app-header-nav-bar/app-header-nav-bar";
 import { CalculatorRootState } from "@stores/calculator-store";
+import Navbar from "./components/navbar/navbar";
```

Comment:
Dachte wir wollten die components loswerden?

Reply:
Das ist eine neue lokale Komponente. Wir wollen die geteilten Komponenten in `/libs/components` loswerden